src/scheduler/task.rs

```diff
@@ -6,6 +6,8 @@
 // http://opensource.org/licenses/MIT>, at your option. This file may not be
 // copied, modified, or distributed except according to those terms.
 
+#![allow(unused_assignments)]
```

Comment:
Why do you need this? Can you move it to he scope/line where you want to turn off the check?

Reply:
You are right, I fixed it